lib/errors/externalServiceError.js

```diff
@@ -1,11 +1,12 @@
 const KuzzleError = require('./kuzzleError');
 
 class ExternalServiceError extends KuzzleError {
-  constructor(message) {
-    super(message, 500);
+  
+  constructor(message, errorName = 'Undocumented error', code = 0) {
```

Comment:
use only the parent class defaults:

```suggestion
  constructor(message, errorName, code) {
```